### DISCUSSION TITLE
[Hooks]: Allows schedule function to receive a timezone option

### DISCUSSION THREAD

**Discussion by Guiqft** [directus/directus#16743] [Feature Requests]
Hi! While working in a custom hook that needs to schedule periodic jobs, I want to be able to specify the timezone for the scheduler, that actually is the default server timezone. 

Looking on source code, looks like `schedule` function is just a proxy for the `schedule` from `node-cron`, that accepts a `timezone` option as a third argument. Can we have this `options` argument on the exposed `schedule` function for hooks too?

With that, we can allows user to set cron jobs on interface for running on the specified daily hours following his local timezone.
Thanks!

**Comment by raphourbe** [comment]
Since you can access packages inside extensions, maybe you can try to import `schedule` from `node-cron` directly inside your hook and work with it? It is not convenient, but it might work :)

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
